FrameworkSystem/scripts/dirac-proxy-init.py

```diff
@@ -36,12 +36,17 @@ def setUploadPilotProxy( self, _arg ):
   def setVOMSExt( self, _arg ):
     self.addVOMSExt = True
     return S_OK()
+  
+  # def setDIRACGroup( self, arg ):
+  #  self.diracGroup = arg
+  #  return DIRAC.S_OK()
 
   def registerCLISwitches( self ):
     ProxyGeneration.CLIParams.registerCLISwitches( self )
     Script.registerSwitch( "U", "upload", "Upload a long lived proxy to the ProxyManager", self.setUploadProxy )
     Script.registerSwitch( "P", "uploadPilot", "Upload a long lived pilot proxy to the ProxyManager", self.setUploadPilotProxy )
     Script.registerSwitch( "M", "VOMS", "Add voms extension", self.setVOMSExt )
+    #Script.registerSwitch( "g:", "group=", "DIRAC Group to embed in the proxy", self.setDIRACGroup )
```

Comment:
What? This is widely used.

Reply:
"g, group=" keys still present and use in dirac-proxy-init